package.json

```diff
@@ -5,36 +5,50 @@
   "keywords": [
     "quill",
     "module",
-    "colaborative",
+    "collaborative",
     "editing"
   ],
-  "author": "Pedro Machado Santa <pedro@reedsy.com>",
+  "contributors": [
+    "Pedro Machado Santa <pedro@reedsy.com>",
```

Comment:
My `pedro@reedsy.com` email is no longer active. Could I kindly ask to update to `pedro.santa@gmail.com` here? Thanks in advance!